src/internal.d.ts

```diff
@@ -58,6 +58,7 @@ export interface Component<P = {}, S = {}> extends preact.Component<P, S> {
 	_prevVNode?: VNode | null;
 	_ancestorComponent?: Component<any, any>;
 	_processingException?: Component<any, any> | null;
+	_pendingError?: Component<any, any> | null;
```

Comment:
Nit: We need to add this to our mangling properties to avoid potential name clashes.

Reply:
Does it matter what name I give it? Was wondering that since I see clashing names in mangle and that worries me to pick one.

Reply:
Mangling this to __E actually adds 3B

Reply:
The default is to mangle `_` properties to single-letter variables, so that makes sense. However, we want to avoid single-letter property names since they're likely to collide.